dotCMS/src/curl-test/PagesResourceTests.json

```diff
@@ -23,7 +23,7 @@
 									"    console.log(jsonData);",
 									"",
 									"    pm.expect(jsonData[\"bundleName\"]).to.eql(\"pages.tar.gz\");",
-									"    pm.expect(jsonData[\"status\"]).to.eql(\"SUCCESS\");",
+									"    pm.expect(jsonData[\"status\"]).to.eql(\"RECEIVED_BUNDLE\");",
```

Comment:
This one should say at `SUCCESS` - It's a known issue

Reply:
Fixed.